src/test/mod.rs

```diff
@@ -20,50 +20,67 @@ use env_logger;
 
 use analysis;
 use build;
-use server as ls_server;
+use server::{self as ls_server, ServerMessage, Request, Method};
 use vfs;
 
 use self::types::src;
 
 use url::Url;
-use ls_types::TextDocumentIdentifier;
+use ls_types::*;
 use serde_json;
 use std::path::{Path, PathBuf};
 
 const TEST_TIMEOUT_IN_SEC: u64 = 10;
 
+impl ServerMessage {
+    pub fn simple_test_initialize(id: usize, root_path: Option<String>) -> ServerMessage {
```

Comment:
I'll rename it. It's `pub` because I also used it [here](https://github.com/rust-lang-nursery/rls/pull/323/files#diff-4ce93534efc34e923ce01e975eb7ed80R626). Should I get rid of that?